client/ayon_houdini/plugins/publish/collect_current_file.py

```diff
@@ -15,23 +15,15 @@ def process(self, context):
         """Inject the current working file"""
 
         current_file = hou.hipFile.path()
-        if not os.path.exists(current_file):
+        if (
+                hou.hipFile.isNewFile()
+                or not os.path.exists(current_file)
```

Comment:
Should we add checking for unsaved changes ?
```suggestion
                or not os.path.exists(current_file)
                or hou.hipFile.hasUnsavedChanges()
```

Reply:
Sorry - I think you misunderstood, and I understand why - because of the log message.

It's **not** about having unsaved changes, that's totally up to you - as long as we know what your filepath should be for your current file. The "unsaved" I'm referring to is the case where no filepath was set at all currently (like a new scene) which Houdini by default returns as `untitled.hip` (but that's also customizable via an env var or something).

Does that change your suggestion - because it seems wrong? :)